src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -752,7 +781,10 @@ OSXOutput::Close() noexcept
 {
 	AudioOutputUnitStop(au);
 	AudioUnitUninitialize(au);
-	delete ring_buffer;
+	if (ring_buffer) {
```

Comment:
Superfluous `nullptr` check. Why is it even here? Nobody ever assigns `nullptr` to this variable.

Reply:
Done